Makefile

```diff
@@ -326,8 +328,7 @@ lint-markdown: ## Lint the project's markdown
 	docker run --rm -v "$$(pwd)":/build$(DOCKER_VOL_OPTS) gcr.io/cluster-api-provider-vsphere/extra/mdlint:0.17.0 -- /md/lint -i _releasenotes .
 
 .PHONY: lint-shell
-lint-shell: ## Lint the project's shell scripts
-	docker run --rm -t -v "$$(pwd)":/build:ro gcr.io/cluster-api-provider-vsphere/extra/shellcheck
+lint-shell: verify-shellcheck ## Lint the project's shell scripts
```

Comment:
Should we get rid of lint-shell?

Reply:
I'd be okay with that 👍

Reply:
Let's do it

Reply:
Also verified we do not use this or the shell script in CI.